docs/tutorials/cloud-foundry.md

```diff
@@ -44,7 +44,7 @@ This document takes us through the steps that will install Move2Kube and use Mov
    ```console
    $ cd move2kube-demos
    ```
-   Let's see the structure inside the `./samples/cloud-foundry` directory.
+   Let's see the structure inside the `./samples/cloud-foundry` directory. The folder contains the source code files and the manifest.yml file.
```

Comment:
Can we possibly add it as the title line of this prerequisite, and then a sample of this is present in this repo? Also we can move this point as the first one.

Reply:
Yes sure, I will update it.